src/js/utils/DOMUtils.js

```diff
@@ -28,21 +29,19 @@ var DOMUtils = {
 
     var computeInnerBound = function (acc, key) {
       var val = parseInt(compstyle[key], 10);
-      if (_.isNaN(val)) {
+      if (Number.isNaN(val)) {
```

Comment:
although `Number#isNaN` is not heavily supported, `isNaN` by itself apparently is

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN